File: 14.桥接模式/geqiangbao_20201201_bridge_swift/geqiangbao_20201201_bridge_swift/桥接模式/案例一原理案例/Size/SmallCoffee.swift
//
//  SmallCoffee.swift
//  geqiangbao_20201201_bridge_swift
//
//  Created by xiaobao on 2020/12/1.
//

import UIKit

class SmallCoffee: Coffee {

    override func makeCoffee() {
        let a = "小杯" + (self.additive?.getContent() ?? "") + "咖啡"
        print(a)
    }
}

File: 14.桥接模式/geqiangbao_20201201_bridge_java/src/桥接模式java/案例一原理案例/Size/SmallCoffee.java
package 桥接模式java.案例一原理案例.Size;

import 桥接模式java.案例一原理案例.Coffee;

public class SmallCoffee extends Coffee {

    @Override
    public void makeCoffee() {
      String a = "小杯" + this.additive.getContent ();
        System.out.println (a);
    }
}
